newswires/client/src/WireItemTable.tsx

```diff
@@ -31,38 +33,63 @@ const fadeOutBackground = css`
 `;
 
 export const WireItemTable = ({ wires }: { wires: WireData[] }) => {
-	const { config, handleSelectItem } = useSearch();
+	const { config, handleSelectItem, loadMoreResults } = useSearch();
+
+	const [isLoading, setIsLoading] = useState<boolean>(false);
 
 	const selectedWireId = config.itemId;
 
+	const handleLoadMoreResults = () => {
+		if (wires.length > 0) {
+			setIsLoading(true);
+
+			const beforeId = Math.min(...wires.map((wire) => wire.id)).toString();
+
+			void loadMoreResults(beforeId).finally(() => {
```

Comment:
this seems like a great way of handling the async state update 🤩

one small suggestion is to maybe rename `isLoading` to `isLoadingMore`, just to fit with the `loadMoreResults` naming and differentiate it from the global `loading` status? But this isn't a blocking suggestion

Reply:
I agree, I nearly used that name :)